libweb3jsonrpc/UnixSocketServer.h

```diff
@@ -23,10 +23,11 @@ along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
 #if !defined(_WIN32)
 
-#include <string>
+#include "IpcServerBase.h"
+#include <atomic>
 #include <thread>
+#include <string>
```

Comment:
I believe "t" is after "s" in most latin-based alphabets, but let it be.

Reply:
I didn't know you follow that ordering.

Reply:
It looks more ordered to me.

Reply:
Yeah, I thought `<string>` doesn't probably include `<thread>`, but `<thread>` might include `<string>`.